programs/relayer/build.rs

```diff
@@ -1,6 +1,12 @@
+use std::env;
+use std::path::PathBuf;
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+
     tonic_build::configure()
         .build_server(true)
+        .file_descriptor_set_path(out_dir.join("relayer_descriptor.bin"))
```

Comment:
What does this do? 😅

Reply:
enables `tonic_build` to output the file descriptor set (file that describes the proto output)

Reply:
I see, but it is only setting the path, not actually generating anything

Reply:
yes, the compile_protos is doing the actual generation